src/WeatherRequest.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+
+const currentWeatherURI = 'https://api.openweathermap.org/data/2.5/weather';
+const hourlyWeatherURI =
+  'https://pro.openweathermap.org/data/2.5/forecast/hourly';
+const dailyWeatherURI =
+  'https://api.openweathermap.org/data/2.5/forecast/daily';
+
+const requestLonLat = async function requestWeatherByLonAndLat(URIparams, URI) {
+  const response = await fetch(
```

Comment:
I would recommend using Axios for the API calls since it already parses the response into JSON and so there is no need to do `response.json()`. Axios also provide other helpful tools and is overall more simple and straightforward than fetch. Although for the sake of this project, I don't know if we'll be using the other things that Axios provides but it would still be good to have at our disposal. You need to do a `import axios from 'axios'`

Reply:
That's a good point! I imported the axios module to the project and switched the calls to using axios.